src/main/target/OMNIBUSF4/target.h

```diff
@@ -174,6 +174,10 @@
 
 #define AVOID_UART1_FOR_PWM_PPM
 
+#define SPEKTRUM_BIND
+// USART1 RX
+#define BIND_PIN                PA10
```

Comment:
thought you might ask :) the pinout description is here: https://github.com/betaflight/betaflight/wiki/Board---OMNIBUSF4#serial-rx